.ai/skills/backend-guidelines/references/guide.md

```
@@ -304,14 +304,14 @@ app/
 |-----------|------|
 | `spatie/laravel-data` | DTOクラス、Inertia連携 |
 | `spatie/laravel-typescript-transformer` | PHP→TypeScript型生成 |
-| `tightenco/ziggy` | フロントエンドへのルート共有 |
+| `laravel/wayfinder` | 型付きroute / actionの生成 |
 
 ### Frontend
 | パッケージ | 用途 |
```

markdownlint-cli2 (0.23.2)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)